velox/substrait/VeloxToSubstraitExpr.cpp

```diff
@@ -158,7 +156,24 @@ const ::substrait::Expression& VeloxToSubstraitExprConvertor::toSubstraitExpr(
         typeConvertor_->toSubstraitType(arena, callTypeExpr->type()));
 
   } else {
-    VELOX_NYI("Unsupported function name '{}'", functionName);
+    if (callTypeExpr->inputs().size() % 2 != 1) {
+      VELOX_NYI(
+          "Number of arguments are always going to be odd for if/then or switch expression");
```

Comment:
Is this assumption accurate? Switch statement has an even number of input when "else" branch is not specified (in this case it is assumed "else null").

Reply:
updated

Reply:
Updated